### PR TITLE
Add definition for 'boolean' in Spanish and fixed typos in the same term

### PR DESCRIPTION
**Pull request by Eirinits**
## Author: 

- Eirini Tsirvouli

## Language: 

- Spanish

## Terms defined:

- Boolean

NB! This  is par

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -961,7 +961,7 @@
     term: "Boolean"
     def: >
       Relating to a variable or data type that can have either a logical value of
-      [true](#true) or [false](#false). Named for George Boole, a 19th
+      [true](#true) or [false](#false). Named after George Boole, a 19th
```
Comment (baileythegreen) [inline glossary.yml:964]: The current version is not incorrect; the two usages are interchangeable in English.

File: glossary.yml [hunk 2]
```diff
@@ -978,11 +978,18 @@
   pt:
     term: "Booleano"
     def: >
-      Relacionado a uma uma variável ou tipo de dado que pode ter um valor lógico de
+      Relacionado a uma variável ou tipo de dado que pode ter um valor lógico de
```
Comment (baileythegreen) [inline glossary.yml:981]: Good.

**Review by baileythegreen — COMMENTED** [review]
Thanks for contributing!

I've commented on the the two small changes.

I'll direct this to the Spanish team for approval.

**Comment by baileythegreen** [comment]
@fmichonneau Should I be able to direct this to the Spanish team? Or is that a maintainer-level privilege?

**Comment by fmichonneau** [comment]
you should be able to assign it to the carpentries/glosario-editors-es team (not tagging them here to avoid double notification)

**Comment by baileythegreen** [comment]
@fmichonneau Got it! I looked in the right place, I just apparently didn't go all the way to the bottom!

**Review by ian-flores — APPROVED** [review]

**Comment by elletjies** [comment]
@all-contributors please add @Eirinits for translation

**Pull request merged**